web_development_101/project_html_css.md

```diff
@@ -1413,7 +1413,7 @@ Submit a link below to the github repo with your files in it by using a pull req
 * [Faris Ibrahim's Solution (Easy)](https://github.com/procusr/google-homepage) - [View in Browser](https://procusr.github.io/google-homepage/)
 * [Giorgi Shengelaia's Solution (Easy)](https://github.com/supuun/Google_Clone) - [View in Browser](https://supuun.github.io/Google_Clone/)
 * [3lux's solution (Easy)](https://github.com/3lux/google-homepage) - [View in browser](https://3lux.github.io/google-homepage/)
-* [Rudi Boshoff's Solution (Easy)](https://rudiboshoff.github.io/google-homepage/) - [View in browser](https://rudiboshoff.github.io/google-homepage/)
+* [Rudi Boshoff's Solution (Easy)](https://rudiboshoff.github.io/google-homepage/) - [View in browser](https://rudiboshoff.github.io/google-homepage/) - [Rudi Boshoff's Solution (Difficult)](https://github.com/RudiBoshoff/google-search-results) - [View in Browser](https://rudiboshoff.github.io/google-search-results/)
```

Comment:
Thank you for your response on my suggestion about the instructions. Please could you also accept my changes to add my difficult solution to the overall document. Thank you again

Reply:
Hi @RudiBoshoff 

To add your solution to this without the instructions you'll have to open a new pull request with just the addition.